src/db/sqlite/lrsql/sqlite/sql/delete.sql

```diff
@@ -59,3 +59,46 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope
+
+----------------------begin components of delete-actor-----
+-- :name delete-actor-st2st
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement 
+WHERE ancestor_id IN (SELECT statement_id FROM statement_to_actor
+WHERE actor_ifi = :actor-ifi)
+OR descendant_id IN (SELECT statement_id FROM statement_to_actor
+WHERE actor_ifi = :actor-ifi)
```

Comment:
I would like the re-formatting of the SQL delete ops to also apply to SQLite. I _thought_ this was obvious when I pointed it out in Postgres, but I guess not, so my apologies.

Reply:
changed